sdcflows/workflows/b0.py

```diff
@@ -0,0 +1,91 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+
+
+"""
+
+from nipype.pipeline import engine as pe
+from nipype.interfaces import utility as niu, fsl, ants
+from nipype.workflows.dmri.fsl.utils import demean_image, cleanup_edge_pipeline
+from niworkflows.engine.workflows import LiterateWorkflow as Workflow
+from niworkflows.interfaces.bids import DerivativesDataSink
+from niworkflows.interfaces.images import IntraModalMerge
+from niworkflows.interfaces.masks import BETRPT
+from ..interfaces.fmap import FieldEnhance
+
+
+def init_prepare_magnitude_wf(omp_nthreads, name='magnitude_wf'):
```

Comment:
This could be a stupid question, but wouldn't it make sense to just calculate the phasediff map and then reuse the existing phasediff (or the fieldmap, if the output of this is in Hz) workflows?

Reply:
That happens here: https://github.com/mattcieslak/sdcflows/blob/phase1phase2/sdcflows/workflows/phdiff.py#L30.